File: php-laraval-app/app/Models/Translation.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Relations\MorphTo;
use Illuminate\Database\Eloquent\SoftDeletes;

class Translation extends Model
{
    use SoftDeletes;

    /**
     * Indicates if the model should be timestamped.
     *
     * @var bool
     */
    public $timestamps = true;

    /**
     * The attributes that are mass assignable.
     *
     * @var array
     */
    protected $fillable = [
        'translationable_type',
        'translationable_id',
        'key',
        'en_US',
        'nn_NO',
        'sv_SE',
    ];

    /**
     * The attributes that should be cast to native types.
     *
     * @var array
     */
    protected $casts = [
        'created_at' => 'datetime',
        'updated_at' => 'datetime',
        'deleted_at' => 'datetime',
    ];

    /**
     * Define the columns that always be returned.
     *
     * @var string[]
     */
    protected array $includes = ['id', 'key', 'en_US', 'nn_NO', 'sv_SE'];

    /**
     * Define the models for polymorphic relations.
     *
     * @var array<string,string[]>
     */
    protected array $relationsMorphMap = [
        'translationable' => [
            Service::class,
            Addon::class,
            Product::class,
            Category::class,
            GlobalSetting::class,
            CustomTask::class,
            LaundryPreference::class,
            Option::class,
            ProductCategory::class,
            PropertyType::class,
        ],
    ];

    /**
     * Get the parent.
     */
    public function translationable(): MorphTo
    {
        return $this->morphTo('translationable');
    }
}

File: dotnet/src/Downstairs.Infrastructure/Persistence/Models/Translation.cs
namespace Downstairs.Infrastructure.Persistence.Models;

public partial class Translation
{
    public long Id { get; set; }

    public string TranslationableType { get; set; } = null!;

    public long TranslationableId { get; set; }

    public string Key { get; set; } = null!;

    public string? EnUs { get; set; }

    public string? NnNo { get; set; }

    public string? SvSe { get; set; }

    public DateTime? CreatedAt { get; set; }

    public DateTime? UpdatedAt { get; set; }

    public DateTime? DeletedAt { get; set; }
}